src/examples/src/widgets/card/ActionButtons.tsx

```diff
@@ -0,0 +1,24 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Card from '@dojo/widgets/card';
+
+import Button from '../../../../button/index';
+import * as cardCss from '../../../../theme/card.m.css';
+
+const factory = create();
+
+export default factory(function ActionButtons() {
+	return (
+		<div styles={{ width: '400px' }}>
+			<Card
+				actionsRenderer={() => (
+					<div classes={cardCss.actionButtons}>
+						<Button onClick={() => console.log('action clicked.')}>Action</Button>
```

Comment:
I think we are trying to render something for the user over having something logged to the console.

Reply:
Fair enough, this was just a lift-and-shift of the old examples. I agree that updating to show a real result on the page would be better.